graphrag_sdk/ontology.py

```diff
@@ -131,22 +131,23 @@ def from_kg_graph(graph: Graph, sample_size: int = 100,):
             l = lbls[0]
             attributes = graph.query(
                 f"""MATCH (a:{l}) call {{ with a return [k in keys(a) | [k, typeof(a[k])]] as types }} 
-                WITH types limit {sample_size} unwind types as kt RETURN kt, count(1)""").result_set
-            ontology.add_entity(Entity(l, [Attribute(attr[0][0], attr[0][1]) for attr in attributes]))
+                WITH types limit {sample_size} unwind types as kt RETURN kt, count(1) ORDER BY kt[0]""").result_set
+            ontology.add_entity(Entity(l, [Attribute(attr[0][0], 'number' if attr[0][1] == 'Integer' or attr[0][1] == 'Float' else attr[0][1].lower()) for attr in attributes]))
```

Comment:
What about types which aren't supported by the SDK? e.g. `Array`, `Map`, `Vector` ?
I think this conversion should be encapsulated within a function.